modules/generic-scala3/src/main/scala-3/pureconfig/generic/derivation/HintsAwareConfigReaderDerivation.scala

```diff
@@ -0,0 +1,18 @@
+package pureconfig
+package generic
+package derivation
+
+import scala.deriving.Mirror
+
+trait HintsAwareConfigReaderDerivation
+    extends HintsAwareCoproductConfigReaderDerivation,
+      HintsAwareProductConfigReaderDerivation:
```

Comment:
General comment: as with the Scala 3 derivation in core, I strongly prefer if all code in this repository uses the same style for scoping. Can you please use braces instead of just indentation to mark all scopes (class, function, conditionals, `match`...)?